openapi/p4pa-pagopa-payments.openapi.yaml

```
@@ -135,6 +135,39 @@ paths:
           description: Invalid request
         "500":
           description: Internal server error
+  /paymentsReporting/{organizationId}/upload:
+    get:
+      tags:
+        - payments-reporting
+      operationId: uploadOfPaymentReporting
```

it's not an upload request, but a fetch
```suggestion
  /paymentsReporting/{organizationId}/fetch:
    get:
      tags:
        - payments-reporting
      operationId: fetchPaymentReporting
```